performance_manager/lib/rt_vehicle_positions.py

```diff
@@ -61,10 +54,13 @@ def transform_vp_dtyes(vehicle_positions: pandas.DataFrame) -> pandas.DataFrame:
         vehicle_positions["start_date"]
     ).astype("Int64")
 
-    # store current_stop_sequence as Int64 [nullable]
-    vehicle_positions["current_stop_sequence"] = pandas.to_numeric(
+    # store current_stop_sequence as stop_sequence Int64 [nullable]
+    vehicle_positions["stop_sequence"] = pandas.to_numeric(
         vehicle_positions["current_stop_sequence"]
     ).astype("Int64")
+    vehicle_positions = vehicle_positions.drop(
+        columns=["current_stop_sequence"]
+    )
```

Comment:
Changed column name to work with `add_event_hash_column` function.

Reply:
weird that there isn't built in functionality to change the name.

Reply:
There is, that would be more memory efficient so I will incorporate